DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: nfidd
 Title: Material to support course on nowcasting and forecasting of infectious
     disease dynamics
-Version: 0.0.0.9999
+Version: 0.1.0.9999
```

Comment:
shall we release the current version on `main` as 0.1.0 then before merging this?